### DISCUSSION TITLE
V3 Night Mode

### DISCUSSION THREAD

**Discussion by bodgitandco** [gtxaspec/wz_mini_hacks#245] [Q&A] [answered]
Been running three V2s for a year off internet with Darfang firmware and motioneye with no problems until 1 of the cameras went down, I replaced it with a V3 without doing my homework!!! no Darfang for V3s.

After much looking around I came across wz_mini_hacks, with the help of the readme and discussions area I reflashed wyze demo_wcv3_4.36.2.5.bin and istalled wz_mini_hacks all with no problem ( Thank you everybody).

When I take the camera off internet every thing works with the exception of Night Vision Mode, it appears that for this to work there needs to be a connection to wyze or am I missing something (entirly possible) in the setup.

Any help or Info appreciated.

**Comment by gtxaspec** [comment, marked as answer]
you can manually enable night mode via command line

`cmd irled on`

**Reply by bodgitandco** [reply]
Wow that was quick.

Just tried it, works a treat. So i guess the next step is to put it in a cron job to switch On/Off every night.

Thank you.

**Comment by LyrixCaz** [comment]
I know this post is a bit old, but i just couldn't find this information anywhere else.. Is there some type of manual or help section for the _cmd_ commands on the v3? after i upgraded to the latest i couldn't edit the camera core settings (like night vision) they were greyed out. But this command worked.

**Reply by archandanime** [reply]
Here you are: https://github.com/gtxaspec/wz_mini_hacks/wiki/Device-Commands

**Reply by LyrixCaz** [reply]
thank you!
